### ISSUE TITLE
Add agn/age/agg/ag- commands to create graphs

### ISSUE THREAD

**Issue by radare** [radare/radare2#2747]
```
ag- . reset core->graph
agn title body  # body can be base64 if prefixed with base64:
age title1 title2 # edge
agg # print core->graph
```

core->graph must be RGraph, and core/graph must be refactored to use util/graph

**Comment by radare** [comment]
cc @ret2libc

**Comment by ret2libc** [comment]
core/graph refactored to use util/graph: done! (in b279f282bae6cfc6101c313efcbc2bea19bdaf8b)

**Comment by radare** [comment]
Awesome!

**Comment by jvoisin** [comment]
♥

**Comment by ret2libc** [comment]
TODO:
* allow spaces in title and body (wrapped with "...")
* body can be base64

**Comment by ret2libc** [comment]
Completely fixed with 6d5d7a1fc965d96284fe0c260489bf0143f53c8a
